src/components/alerting/AlertsPage.tsx

```diff
@@ -0,0 +1,310 @@
+import {
+  AlertResource,
+  alertSeverityOrder,
+  alertState,
+  alertURL,
+  devAlertURL,
+  fuzzyCaseInsensitive,
+} from '../utils';
+import { severityRowFilter } from '../alerting';
+import { sortable } from '@patternfly/react-table';
+import { Helmet } from 'react-helmet';
+import { DropdownItem } from '@patternfly/react-core';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { Link } from 'react-router-dom';
+import KebabDropdown from '../kebab-dropdown';
+
+const tableAlertClasses = [
+  'pf-u-w-50 pf-u-w-33-on-sm', // Name
+  'pf-m-hidden pf-m-visible-on-sm', // Severity
+  '', // State
+  'pf-m-hidden pf-m-visible-on-sm', // Source
+  'dropdown-kebab-pf pf-c-table__action',
+];
+
+const AlertsPage_: React.FC<AlertsPageProps> = () => {
+  const { t } = useTranslation('plugin__monitoring-plugin');
+  const { isDev, alertsKey, defaultAlertTenant } = usePerspective();
+
+  const {
+    data,
+    loaded = false,
+    loadError,
+  }: Alerts = useSelector(({ observe }: RootState) => observe.get(alertsKey) || {});
+  const silencesLoadError = useSelector(
+    ({ observe }: RootState) => observe.get('silences')?.loadError,
+  );
+
+  const alertAdditionalSources = React.useMemo(
+    () => getAdditionalSources(data, alertSource),
+    [data],
+  );
+
+  let rowFilters: RowFilter[] = [
+    // TODO: The "name" filter doesn't really fit useListPageFilter's idea of a RowFilter, but
+    //       useListPageFilter doesn't yet provide a better way to add a filter like this
+    {
+      filter: (filter, alert: Alert) =>
+        fuzzyCaseInsensitive(filter.selected?.[0], alert.labels?.alertname),
+      filterGroupName: '',
+      items: [],
+      type: 'name',
+    } as RowFilter,
+    {
+      defaultSelected: [AlertStates.Firing],
+      filter: (filter, alert: Alert) =>
+        filter.selected?.includes(alertState(alert)) || _.isEmpty(filter.selected),
+      filterGroupName: t('Alert State'),
+      items: [
+        { id: AlertStates.Firing, title: t('Firing') },
+        { id: AlertStates.Pending, title: t('Pending') },
+        { id: AlertStates.Silenced, title: t('Silenced') },
+      ],
+      reducer: alertState,
+      type: 'alert-state',
+    },
+    severityRowFilter(t),
+    {
+      defaultSelected: [defaultAlertTenant],
+      filter: (filter, alert: Alert) =>
+        filter.selected?.includes(alertSource(alert)) || _.isEmpty(filter.selected),
+      filterGroupName: t('Source'),
+      items: [
+        { id: AlertSource.Platform, title: t('Platform') },
+        { id: AlertSource.User, title: t('User') },
+        ...alertAdditionalSources,
+      ],
+      reducer: alertSource,
+      type: 'alert-source',
+    },
+  ];
+
+  if (isDev) {
+    rowFilters = rowFilters.filter((filter) => filter.type !== 'alert-source');
+  }
+
+  const [staticData, filteredData, onFilterChange] = useListPageFilter(
+    data?.filter((alert: Alert) => (isDev ? alertSource(alert) === AlertSource.User : true)),
```

Comment:
This will prevent admins from using the dev console to see the namespace alerts. Although this behavior could be revised, at this stage, we need to maintain parity with the current dev console alerts.

Reply:
Removed the filter here. While testing I got mixed up by the console returning non-namespaced platform alerts while in "no authentication" mode for the rules tenancy api with a namespace, but after some more testing that is how it worked before this change as well. It should now have parity with the dev perspective alerts.